src/app/components/dropdown/dropdown.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+
+interface Options {
```

Comment:
You can try to extract all the interfaces into a separate file and just export it.

Reply:
You could create a interfaces folder